proto/substrait/algebra.proto

```diff
@@ -833,6 +853,12 @@ message Expression {
     }
   }
 
+  enum WindowType {
+    WINDOW_TYPE_UNSPECIFIED = 0;
+    WINDOW_TYPE_ROWS = 1;
+    WINDOW_TYPE_RANGE = 2;
+  }
+
```

Comment:
Just a small suggestion. I think using `window frame` is more proper and more clear to user than using `window type`. Right?
```
  enum WindowFrame {
    WINDOW_FRAME_UNSPECIFIED = 0;
    WINDOW_FRAME_ROWS = 1;
    WINDOW_FRAME_RANGE = 2;
  }
```

Reply:
Thanks for you suggestion. And very sorry that I did not reply in time. I have already changed to WindowFrame.